spec/spec.cue

```diff
@@ -0,0 +1,50 @@
+package v1alpha1
+
+import "time"
+
+#spec: {
+	// list?
+	service: string
+	// optional
+	// TODO: 1050 exceeds go regex limit (1000)
+	description?: string
```

Comment:
Can't use regex to validate 1050 char limit here due to go regex package limit (1000).